cli/cache/mod.rs

```diff
@@ -280,10 +282,11 @@ impl Loader for FetchCacher {
     source: &str,
     module_info: &deno_graph::ModuleInfo,
   ) {
```

Comment:
This function is only called for JSR specifiers that have module info stored in the metadata.